kolibri/core/assets/src/navigation.vue

```diff
@@ -42,59 +21,14 @@
           title: 'Kolibri',
           home_link: '/',
         },
-        userNavDisplay: false,
```

Comment:
Based off of what the visual design team seems to be proposing, that's the plan. However, they're still in the design phase so this might change.